python/sglang/srt/two_batch_overlap.py

```diff
@@ -379,9 +449,17 @@ def prepare_raw(
 
         tbo_split_token_index = cls._compute_split_token_index(batch)
 
+        is_enable_two_chunk = (
+            batch.forward_mode == ForwardMode.EXTEND
+            and batch.extend_seq_lens_cpu is not None
+            and tbo_split_token_index
+            != sum(batch.extend_seq_lens_cpu[: batch.tbo_split_seq_index])
```

Comment:
hmm wondering whether this condition can be improved a bit, e.g. shall we have sth like

```
def _is_two_chunk_split_enabled(extend_lens):
    vanilla_split_seq_index = _split_array_by_balanced_sum(extend_lens)
    left_sum = sum(arr[:vanilla_split_seq_index])
    overall_sum = sum(arr)
    ... original logic ...
```

(note that we will waste some cpu recompute in this case, but that is too tiny to be visible)

the benefit of this function signature is that, we are clear whether two chunk overlap is enabled given a extend_lens, without the need to compute any others
and here, we can change `tbo_split_token_index            != sum(batch.extend_seq_lens_cpu[: **batch.tbo_split_seq_index])` to `_is_two_chunk_split_enabled(batch.extend_lens)` etc